### PR TITLE
Completed react-hot-button-component.

### PR DESCRIPTION
**Pull request by stefancarrera**
react clicker

### CODE REVIEW

File: react-hot-button-component/src/hot-button.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    this.state = { count: 0 };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ count: this.state.count + 1 });
+  }
+
+  render() {
+    if (this.state.count < 3) {
```
Comment (Zhiwutian) [inline react-hot-button-component/src/hot-button.jsx:15]: So while this works, we can do a cleaner job! Try moving the logic for choosing the classname to a separate method, have it return the proper classname, then set that classname as the button classname. That way render only has to return a single button with the proper class.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
good initial solution, now try cleaning up the render method!

**Comment by stefancarrera** [comment]
pickClass

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**